integrations/gradle-bloop/src/main/scala/bloop/integrations/gradle/model/BloopConverter.scala

```diff
@@ -592,11 +592,15 @@ final class BloopConverter(parameters: BloopParameters) {
           val opts = scalaCompileTask.getScalaCompileOptions
           val options = optionList(opts)
           val compilerName = parameters.compilerName
+          val compileOrder =
+            if (!sourceSet.getJava.getSourceDirectories.isEmpty) JavaThenScala
```

Comment:
Couldn't find any official documentation about this, but looks like this SO defines the behaviour: https://stackoverflow.com/questions/23261075/compiling-scala-before-alongside-java-with-gradle

It's weird that the default in Gradle when java sources are defined is `JavaThenScala`. In build tools such as sbt the default is always mixed. Anyhow, I'm trusting this is fine 👍